tests/playwright/integration-tests/log-in.spec.ts

```diff
@@ -22,7 +22,7 @@ test.describe('Log in to application', () => {
   })
 
   test.describe('Given the user is not authenticated', () => {
-    test.use({ storageState: '.auth/unauthenticated-user.json' })
+    test.use({ extraHTTPHeaders: { test: 'header' } })
```

Comment:
what does the `{ test: 'header' }` part do? Is it to override the auth override header?

Reply:
yes to override the auth overrride headers..gives a linting error if i just do  test.use({ extraHTTPHeaders: {} }

Reply:
Ok, I think it would be easier to read and understand for future peeps if we explicitly specify the header that we're overriding and why, something like :
```
 test.use({ extraHTTPHeaders: { 'Authorization': 'unauthenticated user' } })
```

Reply:
sorted